scrapd/core/date_utils.py

```diff
@@ -17,6 +17,18 @@ def is_posterior(d1, d2):
     return parse_date(d1) < parse_date(d2)
 
 
+def clean_date_string(date):
+    """
+    Parse the date from an unspecified format to the specified format.
+
+    :param str date:
+    :return: a date string in the uniform %d/%m/%Y format.
+    :rtype: str
+    """
+    dt = parse_date(date)
+    return datetime.datetime.strftime(dt, "%d/%m/%Y")
```

Comment:
I apologize I made a mistake, the English format for the date is `MM/DD/YYYY`, so if you can just reverse these two first fields that would be great.

That would also make your test pass.